lib/Software/License/ISC.pm

```diff
@@ -14,8 +14,7 @@ __DATA__
 __LICENSE__
 The ISC License
 
-Copyright (c) 2004-2011 by Internet Systems Consortium, Inc. ("ISC")
-Copyright (c) 1995-2003 by Internet Software Consortium
+Copyright (c) {{$self->year}} by {{$self->holder}}
```

Comment:
I saw that some other license files and Leon's suggestion used `_dotless_holder`. Is this really what we are supposed to do? Why not promote that method to a public one if it's the best practice?